go.mod

```diff
@@ -1,8 +1,6 @@
 module sigs.k8s.io/metrics-server
 
-go 1.23.0
-
-toolchain go1.23.1
```

Comment:
Please leave toolchain

Reply:
`go mod tidy` automatically removes it.

Reply:
In the old version it was kept because it was newer than the Go version.